azure-pipelines.yml

```diff
@@ -0,0 +1,71 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - "*"
+
+pr: none
+
+workspace:
+  clean: all
+
+variables:
+- name: SolutionBaseName
+  value: '__SolutionBaseName__'
+- name: BuildPlatform
+  value : 'any cpu'
+- name: BuildConfiguration
+  value: 'release'
+- group: 'Prod Management Resources'
```

Comment:
Is something dependent on the variables in this group?  Scoping a Prod variable group will create problems when we move to yaml deployments.  It should be removed but if it can't a separate variable group should be created that only contains the required variables

Reply:
Done